doc/yarn.md

```diff
@@ -125,6 +125,16 @@ The _start-server.sh_ script does not contain a ```--master``` option. In some c
 --master yarn-client
 ```
 
+### Modifying the &lt;environment&gt;.sh script
+Replace `MANAGER_*` variables with
+```
+MANAGER_JAR_FILE="$appdir/spark-job-server.jar"
```

Comment:
I wonder if it might be good to put these alternatives in `setenv.sh` as well, after all the Mesos one is there but YARN isn't.

Reply:
Good point, but at this point, I don't have any variable which tells me the current spark.submit.deployMode. I do have a path to *.conf file and somehow within BASH, I can try to parse this. What do you say ?
Also @derSascha  any comments?